src/installer/tests/TestUtils/TestProjectFixture.cs

```diff
@@ -75,43 +76,45 @@ public void Dispose()
             }
         }
 
+#nullable enable
+
+        private readonly static object? s_buildFilesLock = new object();
```

Comment:
object?  can this be null?

Reply:
Fallout from an earlier attempt to do a clever thing with Interlocked.CompareExchange, but it turns out some assemblies might call this with different testartifact paths